17.md

```diff
@@ -123,6 +123,11 @@ This NIP offers the following privacy and security features:
 
 The main limitation of this approach is having to send a separate encrypted event to each receiver. Group chats with more than 100 participants should find a more suitable messaging scheme.
 
+## NIP-19 entity
+To create an `nchat` link:
+  - `kind`: `17`
```

Comment:
Correct kind is 14

Reply:
`kind` is NIP number not event kind

Reply:
Why use the NIP number instead of just the chat kind?

Reply:
Yea maybe it makes more sense to use the kind number, originally i had the HRP as `nchat17` and `nchat28` but decided to change it last minute to use the kind entry.